src/commands/ref_command_handler.js

```diff
@@ -34,16 +33,24 @@ function cppReference(message, args) {
                 }
             }
 
-
             const results = [];
             for (const ref of hrefs) {
                 if (ref.search(args[0]) > 0) {
                     console.log(ref);
                     results.push("https://en.cppreference.com/" + ref);
                 }
             }
+			
             if (results.length > 0) {
-                channel.send(results);
+				var l = 0;
```

Comment:
Please use actual variable names, `l` means nothing to me.
```suggestion
				let stringLength = 0;
```

Reply:
Yeah, these are probably (bad) habits from programming in C. I have to remeber that high-level languages use sweet variable names :)